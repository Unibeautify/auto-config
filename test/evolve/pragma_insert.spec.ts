import { testGenetic } from "../testGenetic";
import prettier from "@unibeautify/beautifier-prettier";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("pragma_insert", {
  beautifiers: [prettier, jsBeautify],
  language: "JavaScript",
  originalText: `console.log("hello world");`,
  desiredText: `/** @format */\n\nconsole.log('hello world')\n`,
});
