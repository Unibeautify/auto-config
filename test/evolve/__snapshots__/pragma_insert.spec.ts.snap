// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pragma_insert: beautifiedText 1`] = `
/** @format */

console.log('hello world')

`;

exports[`pragma_insert: entity 1`] = `
Object {
  "options": Object {
    "beautifiers": Array [
      "Prettier",
    ],
    "end_with_semicolon": false,
    "pragma_insert": true,
    "quotes": "single",
  },
}
`;

exports[`pragma_insert: fitness 1`] = `2`;
